command.js

```diff
@@ -26,7 +26,7 @@ const yargs = require('yargs')
     default: defaults.infile
   })
   .option('message', {
-    alias: 'm',
+    alias: ['m', 'releaseCommitMessageFormat'],
```

Comment:
maybe add a note that we'll deprecate message in the next `major`, and warn if people use `--message` or `-m`?